src/selectors/partners/partner.js

```diff
@@ -1,25 +1,17 @@
 
 import find from 'lodash/find';
 import { createSelector } from 'reselect';
-import partners from 'json/partners.json';
 
 // Get partner path
 const partnerPath = state => state.routing.locationBeforeTransitions.pathname;
 
-// Get partner data
-const getPartnerData = (_path) => {
-  const name = _getPartnerName(_path);
-  const partner = partners.filter(p => p.name === name)[0] || {};
-  return partner;
-};
-
 // Get partner path and get its name from it
-const _getPartnerName = (_path) => {
-  const pathArray = _path.split('/');
-  const partnerName = decodeURIComponent(pathArray[pathArray.length - 1]);
+const getPartnerId = (_partnerPath) => {
```

Comment:
I need to know the purpose of this entire file...

You have 2 options instead of creating a selector only to get the id of the partner:
- Use the hooks on the router onEnter and/or onChange (see explore page)
- Use the this.props.params.id...

I prefer the first one, and you should change it in all pages that have the same issue